tapir/coop/templates/coop/welcome_desk_member.html

```diff
@@ -0,0 +1,57 @@
+{% extends "accounts/base.html" %}
```

Comment:
Shouldn't this template also contain a box with the shift status as the member office sees it?

Maybe we could actually leverage a CSS grid layout here. I was sort of expecting this page to look somewhat similar to the normal member page, but with a sidebar on the right with the messages. Then we could place the boxes in the same place always.

Reply:
By shift status you mean ABCD/Flying? Or if they have negative/positive shift credits?
I'm not sure why the welcome desk would need that? 

If we do something similar to the member page, you mean we'd do similar cards? What would we put in them?

Reply:
My idea would be that the welcome desk can do shift-related tasks, like answering when their next shift is and registering them for a shift. My thinking is that it makes the job more interesting, allows us to fill upcoming empty shifts and gives people who are not so much into computers access to these features by just talking to the welcome desk. 

We can submit like this though and deal with it in a follow-up PR.